features/steps/fhir.py

```diff
@@ -144,7 +145,8 @@ def step_impl(context):
 @given('there is at least 1 entry')
 def step_impl(context):
     resource = context.response.json()
-    entries = resource.get('entry', [])
+    entries = [entry['resource'] for entry in resource.get('entry', [])
```

Comment:
Thanks! Yes, I'd start with these two tests, since they give us insight into which sample data might be missing. (Let me know if you think there are other important targets!)

I think making the logic explicit in the feature language would be perfect and straightforward (at the expense of having to update 10ish individual feature files).